front/src/styles/common.css

```diff
@@ -4,39 +4,68 @@
   font-weight: 600;
   font-size: 14px;
   font-family: Open Sans;
+  font-style: normal;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 h1 {
   font-size: 54px;
+  line-height: 92px;
 }
 
 h2 {
   font-size: 38px;
+  line-height: 63px;
 }
 
 h3 {
   font-size: 33px;
+  line-height: 53px;
 }
 
 h4 {
   font-size: 23px;
+  line-height: 34px;
 }
 
 h5 {
   font-size: 18px;
+  line-height: 27px;
 }
 
 h6 {
   font-size: 15px;
+  line-height: 21px;
 }
 
-.v-btn {
+body{
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.text-small{
+ font-size: 12px;
+ line-height: 15px;
+}
+
+.text-xsmall{
+  font-size: 9px;
+  line-height: 11px;
+}
+
+.text-normal{
+  font-weight: normal;
+}
+
+.mmt-btn{
   text-transform: none;
   letter-spacing: normal;
 }
 
+.theme--light.v-card.auth-card, .theme--light.v-card.recovery-card {
+  background-color: #F6F8FB;
```

Comment:
variable?

Reply:
What do you mean? I don't really understand the question